.pre-commit-config.yaml

```diff
@@ -23,6 +13,16 @@ repos:
           args: ["--fix", "--show-fixes", "--exit-non-zero-on-fix", "--ignore=PLR2004,PLR0913"]
           # include: "tests"
           exclude: "^kedro/templates/|^features/steps/test_starter/|kedro"
```

Comment:
Move ruff before black

>  https://github.com/astral-sh/ruff-pre-commit
Ruff's pre-commit hook should be placed after other formatting tools, such as Black and isort, unless you enable autofix, in which case, Ruff's pre-commit hook should run before Black, isort, and other formatting tools, as Ruff's autofix behavior can output code changes that require reformatting.